src/System.Drawing.Common/src/System/Drawing/StringFormat.cs

```diff
@@ -256,7 +256,7 @@ public void SetTabStops(float firstTabOffset, float[] tabStops)
         {
             if (firstTabOffset < 0)
             {
-                throw new ArgumentException(SR.Format(SR.InvalidArgument, "firstTabOffset", firstTabOffset));
+                throw new ArgumentException(SR.Format(SR.InvalidArgumentValue, "firstTabOffset", firstTabOffset));
```

Comment:
If you want you could change this to `nameof(firstTabOffset)`

Reply:
done